volentix/settings.py

```diff
@@ -334,3 +336,18 @@ def celery_base_data_hook(request, data):
     "http": f"http://{CRAWLERA_AUTH_TOKEN}:@{CRAWLERA_PROXY_HOST}:{CRAWLERA_PROXY_PORT}/"
 }
 # EOF crawlera settings
+
+# LogDNA settings
+LOGGING_CONFIG = None
+LOGDNA_TOKEN = config('LOGDNA_TOKEN', default='this-token-does-not-work')
+
+options = {
+    'app': 'venue',
+    'env': config('ENV', default='dev'),
+    'index_meta': True
+}
+handler = LogDNAHandler(LOGDNA_TOKEN, options)
+LOGGER = logging.getLogger('logdna')
+LOGGER.setLevel(logging.DEBUG)
+LOGGER.addHandler(handler)
```

Comment:
Still, do not understand why you don't want to configure this logger in the standard way, but ok.